string DEFAULT=   steering | throttle  | door 
string info
uint16[] channels